test/models/name_test.rb

```diff
@@ -290,7 +290,7 @@ def test_format_string
   end
 
   def test_upper_word_pats
-    pat = /^#{::Name::Parse::UPPER_WORD}$/
+    pat = /^#{::Name::Parse::UPPER_WORD}$/o
```

Comment:
I like having the regular expression right there where it's used, but I feel like the creation of the regex doesn't have to happen each time we run test_upper_word_pats... could it be turned into a (frozen?) global just outside of the method?  Or does the "o" flag basically make the regex a persistent constant that's equivalent to a global?